features/steps/formatting_steps.rb

```diff
@@ -272,7 +272,7 @@
 end
 
 Then(/^I should see the undefined symbold message$/) do
-  run_output.should include(red("⌦  Undefined symbols for architecture x86_64"))
+  run_output.should include(red("❌  Undefined symbols for architecture x86_64"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.